File: exam-slammr-webapp/src/test/java/uk/co/claritysoftware/exam/slammr/webapp/service/model/user/ExamSlammrUserProfileTest.java
package uk.co.claritysoftware.exam.slammr.webapp.service.model.user;

import org.junit.Test;

import nl.jqno.equalsverifier.EqualsVerifier;
import uk.co.claritysoftware.exam.slammr.webapp.service.model.user.ExamSlammrUserProfile;

/**
 * Uuit test class for {@link ExamSlammrUserProfile}
 */
public class ExamSlammrUserProfileTest {

	@Test
	public void shouldHonourEqualsHashcodeContract() {
		EqualsVerifier.forClass(ExamSlammrUserProfile.class)
				.verify();
	}
}

File: exam-slammr-webapp/src/test/kotlin/uk/co/claritysoftware/exam/slammr/webapp/service/model/user/ExamSlammrUserProfileTest.kt
package uk.co.claritysoftware.exam.slammr.webapp.service.model.user

import nl.jqno.equalsverifier.EqualsVerifier
import org.junit.Test

/**
 * Uuit test class for [ExamSlammrUserProfile]
 */
class ExamSlammrUserProfileTest {

	@Test
	fun shouldHonourEqualsHashcodeContract() {
		EqualsVerifier.forClass(ExamSlammrUserProfile::class.java)
				.verify()
	}
}